pkg/slo/slo.go

```diff
@@ -114,7 +184,31 @@ func getPMScoreResult(bugMap bugs.TeamMap, sloData sloAPI.Data, teamInfo teams.T
 	return result
 }
 
-func GetTeamResult(bugMaps map[string]bugs.TeamMap, orgData *teams.OrgData, teamInfo teams.TeamInfo) sloAPI.TeamResult {
+func getCIResult(ciComponentsMap map[string]sippyv1.MinimumPassRatesByComponent, sloData sloAPI.Data, teamInfo teams.TeamInfo) sloAPI.Result {
+	minPass := 100.0
+	for _, c := range teamInfo.Components {
+		passRate, found := ciComponentsMap[c]
+		if !found {
+			continue
+		}
+
+		if pr, ok := passRate.PassRates["latest"]; ok && pr.Percentage < minPass {
+			minPass = pr.Percentage
+		}
+	}
+	obligation := int(sloData.Count)
+	if obligation == 0 {
```

Comment:
We should do defaulting in the shiftzilla config. You should be able to test that locally creating an overwrite yaml file and use --overwrite-team-data on the team exporter.

Reply:
Fixed.

So this means we need a shiftzilla PR first?

Reply:
https://github.com/openshift/li/pull/3994

Reply:
i merged https://github.com/openshift/li/pull/3994

Reply:
merged